tests/test_models.py

```diff
@@ -1,5 +1,6 @@
 """Tests for statistics functions within the Model layer."""
```

Comment:
I'm just gonna try to comment something here, because I don't know what to comment haha